components/page-header/index.jsx

```diff
@@ -35,8 +40,17 @@ const propTypes = {
 	className: PropTypes.string,
 	/**
 	 * The type of component
+	 * Note: Extra options are added to make the version backward compatible
 	 */
-	variant: PropTypes.string,
+	variant: PropTypes.oneOf([
+		'base',
+		'object-home',
+		'record-home',
+		'related-list',
+		'objectHome',
```

Comment:
The only ones should be removed.

Reply:
Removed. I did this to prevent warning in the backward codes otherwise it would have been shown a double warning like the following. Anyways, I had removed it and pushing the changes.

![image](https://user-images.githubusercontent.com/34851451/55150695-b36e9080-5172-11e9-90a6-4b816c88ff9a.png)